Makefile

```diff
@@ -4,3 +4,10 @@ build :
 	rm metr1c
 clean :
 	rm metr1c.tar.gz
+install : build
+	tar -zxvf ./metr1c.tar.gz
+	rm ./metr1c.tar.gz
```

Comment:
Suggest removing the archive generation steps (please see comments to the issue).